modules/Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -409,6 +464,112 @@ private void captureYield(List<MovecraftLocation> harvestedBlocks) {
             }
         }
     }
+    
+    private boolean processNetherPortal(@NotNull Block block) {
```

Comment:
The body of this methodis indented by one space too many, and uses both tabs and spaces.